lib/docs/src/tag.stories.tsx

```diff
@@ -1,61 +1,56 @@
 import { Meta } from "@storybook/react";
-import { Fragment } from "react";
+import { Fragment, useState } from "react";
 import { Button, DivPx, Tag } from "../../core/src";
 import { Utils } from "./utils";
-
-const colors: string[] = [
-	"red",
-	"yellow",
-	"green",
-	"blue",
-	"indigo",
-	"purple",
-	"pink",
-	"gray",
-];
+import { GalleryFeedbackTag } from "../../gallery/src/feedback/tag";
 
 const meta: Meta = {
 	title: "Components/Tag",
 	component: Tag,
 	argTypes: {
 		children: Utils.arg("text"),
-		color: Utils.arg(colors),
+		color: Utils.arg(Object.keys(Tag.colors)),
 	},
 };
 
-Utils.page.component(meta, { sticky: true, shots: [] });
+Utils.page.component(meta, { sticky: true, shots: [<GalleryFeedbackTag />] });
```

Comment:
the <GalleryFeedbackTag /> is used inside an array so it must have a key (React has a warning in the console for this)